conformance/lang/expressions/relational-expr/greater_than_expression.balt

```diff
@@ -0,0 +1,1162 @@
+   io:println(c > b); // @output false
+   io:println(a > d); // @output true
+   io:println(d > a); // @output false
+   io:println(d > b); // @output false
+   io:println(b > d); // @output true
+}
+
+Test-Case: output
+Description: Test greater than expression for user defined sub types of integer built in sub types
+Labels: DecimalNumber, int:Signed8, int:Unsigned32, module-type-defn, relational-expr, union-type
+
+type Ints int:Signed8|int:Unsigned32;
+
+function init() {
+   Ints a = 12;
+   int:Signed8|int:Unsigned32 b = 13;
+
+   io:println(a > b); // @output false
+   io:println(b > a); // @output true
+}
+
+Test-Case: output
+Description: Test greater than expression for positive optional integers
+Labels: DecimalNumber, int, numeric-nil-lifting, optional-type, relational-expr
+
+function init() {
+   int? a = 1;
+   int? b = 101;
+   int? c = 0;
+   int? d = 9223372036854775807;
+   io:println(a > b); // @output false
+   io:println(c > a); // @output false
+   io:println(a > c); // @output true
+   io:println(d > a); // @output true
+   io:println(b > d); // @output false
+   io:println(a > a); // @output false
+}
+
+Test-Case: output
+Description: Test greater than expression for optional negative integers
+Labels: DecimalNumber, int, numeric-nil-lifting, optional-type, relational-expr, unary-minus
+
+function init() {
+   int? a = -1;
+   int? b = -101;
+   int? c = -10;
+   int? d = -9223372036854775807;
+   io:println(a > b); // @output true
+   io:println(c > a); // @output false
+   io:println(a > c); // @output true
+   io:println(d > a); // @output false
+   io:println(b > d); // @output true
+   io:println(a > a); // @output false
+}
+
+Test-Case: output
+Description: Test greater than expression where one operand is negative integer and other one is positive integer,
+            with optional type-descriptors
+Labels: DecimalNumber, int, numeric-nil-lifting, optional-type, relational-expr, unary-minus
+
+function init() {
+   int? a = -1;
+   int? b = 101;
+   int? c = -10;
+   int? d = 9223372036854775807;
+   io:println(a > b); // @output false
+   io:println(b > a); // @output true
+   io:println(d > c); // @output true
+   io:println(c > d); // @output false
+   io:println(a > d); // @output false
+   io:println(d > a); // @output true
+}
+
+Test-Case: output
+Description: Test greater than expression where one operand is an optional integer and other operand is non optional integer
+Fail-Issue: ballerina-platform/ballerina-lang#35195
+Labels: DecimalNumber, int, numeric-nil-lifting, optional-type, relational-expr
+
+function init() {
+   int? a = 1;
+   int b = 101;
+   int? c = 0;
+   int d = 9223372036854775807;
+   io:println(a > b); // @output false
+   io:println(a > d); // @output false
+   io:println(b > c); // @output true
+   io:println(d > c); // @output true
+}
+
+Test-Case: output
+Description: Test greater than expression for optional int:Unsigned8
+Labels: DecimalNumber, int:Unsigned8, relational-expr
```

Comment:
missing `optional-type` label. There are some other places too. Let's check and update.